Задача 1

jacsmedvedev opened the pull request:
Дан массив точек. Каждая точка характеризуется координатами х, у и названием точки. Сортировать точки в массиве по возрастанию удаления расстояния точки от начала координат и вывести их на экран

jacsmedvedev committed:
task_from_teacher_1

В процессе

Commit:
Задача 1.

Дан массив точек. Каждая точка характеризуется координатами х, у и
названием точки. Сортировать точки в массиве по возрастанию удаления
расстояния точки от начала координат и вывести их на экран.

P.S. это ничего что почти все методы и конструкторы копировались с
примера ?

strazhkoe commented on src/task_from_teacher_1/Point_properties.java:
```diff
@@ -0,0 +1,37 @@
+package task_from_teacher_1;
+
+public class Point_properties {
```
Убери нижнее подчеркивание из названия класса

strazhkoe commented on src/task_from_teacher_1/Point_properties.java:
```diff
@@ -0,0 +1,37 @@
+		return y;
+	}
+
+	public void sety(int y) {
```
getX getY setX setY - так должны называться методы

strazhkoe commented on src/task_from_teacher_1/Points.java:
```diff
@@ -0,0 +1,55 @@
+		points[5] = new Point_properties("X", 0, 0);
+
+		bubbleSortx(points);
+		//bubbleSorty(points);
```
Please delete

strazhkoe commented on src/task_from_teacher_1/Points.java:
```diff
@@ -0,0 +1,55 @@
+	public static void bubbleSortx(Point_properties[] points) {
+		for (int i = points.length - 1; i > 0; --i) {
+			for (int j = 0; j < i; ++j) {
+				if ((Math.abs(points[j].getx()) > Math.abs(points[j + 1].getx())
```
Расстояние от точки до начала координат - это корень квадратный из суммы квадратов координат.

jacsmedvedev committed:
task_from_teacher_1

Исправил замечания:
1. переименовал класс Point_properties в PointProperties
2. переименовал get- и set- методы в классе Points
3. переделал формулу расчета расстояния от точки до начала координат

jacsmedvedev committed:
task_from_teacher_1 (repushed)

1.Вставил комменты
2.Создал метод distanceToZero для расчета расстояния в отдельном методе.
3. Прошлый раз запушилось как то криво, без нужных поправок.
